doc/src/Howto_bulk2slab.rst

```diff
@@ -0,0 +1,138 @@
+===========================
+Convert bulk system to slab
+===========================
+
+A regularly encountered simulation problem is how to convert a bulk
+system that has been run for a while to equilibrate into a slab system
+with some vacuum space.  The challenge here is that one cannot just
+change the box dimensions with the :doc:`change_box command
+<change_box>` or edit the box boundaries in a data file because some
+atoms will have non-zero image flags from diffusing around.  Changing
+the box dimensions results in an undesired displacement of those atoms,
+since the image flags indicate how many times the box length in x-, y-,
+or z-direction needs to be added or subtracted to get the "unwrapped"
+coordinates.  By changing the box dimension this distance is changed and
+thus those atoms move unphysically relative to their neighbors with zero
+image flags.  Setting image flags forcibly to zero creates problems because
+that could break apart molecules by have one atom of a bond on the top
+of the system and the other at the bottom.
+
+.. _bulk2slab:
+.. figure:: JPG/rhodo-both.jpg
```

Comment:
The snapshots looks more like a lipid bilayer than a Rhodopsin protein in water to me.

Reply:
Yes, Rhodopsin is a transmembrane protein. I just too the data file from the bench folder.